aptos-move/framework/move-stdlib/src/natives/type_name.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright © Aptos Foundation
```

Comment:
This one is interesting. I don't think we took it over in our source view of the stdlib. We have an alternative `aptos_stdlib::type_info`

Reply:
We should probably check if there are more modules for which we do not have a source and evaluate whether we either add the source or remove the native impl. But can be done as a next step.

Reply:
I removed it